bundles/org.openhab.binding.mybmw/src/main/resources/OH-INF/thing/vehicle-update-channel-types.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mybmw"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<channel-type id="state-update-channel">
+		<item-type>Switch</item-type>
+		<label>trigger the update of the state</label>
+		<state readOnly="false"/>
```

Comment:
```suggestion
		<state readOnly="false"/>
		<autoUpdatePolicy>veto</autoUpdatePolicy>
```

Reply:
The `readOnly` state description is `false` by default, so should be removed.

Reply:
Ok so why do you keep it here and add the autoUpdatePolicy?

Reply:
Ok